### DISCUSSION TITLE
Github act hacked

### DISCUSSION THREAD

**Discussion by TrivediEnterprisesInc** [community/community#17391] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there!

I have been using Github for my private repos for about a year, to host about 2 years of work (about 6000 man hours; startup; tens of thousands of commits).

On or about May 25th I was locked out from my github repo AND both my email accounts; hacked and locked out.  I used VERY strong passwords but they probably had a keyboard intercept.  I have been using a public computer @ a library.

I have set up a temp alt repo to continue work but have NO backups (the email acts were my backup source (gmail Files etc.)

HELP!  What can I do?  I can't afford to lose two years of work.

details->
earlier (compromised) acct: TrivediSolutions, MikeTrivedi
current acct: TrivediEnterprisesInc, MikeTrivedi
current (working) email address: [REDACTED DUE TO PERSONAL INFORMATION]
Non-working (locked out) email addresses: [REDACTED DUE TO PERSONAL INFORMATION]

Thanks in advance,
Mike Trivedi

**Comment by queenofcorgis** [comment, marked as answer]
Hi there,

I'm so sorry to hear this happened!  This space was created to give feedback on GitHub and its products. In this instance and for any account-related issues, I recommend reaching out to our [Support team ](https://support.github.com/request)as they'll be the ones able to assist :octocat: 😄 

Good luck!

**Reply by TrivediEnterprisesInc** [reply]
Hi, Lili!
Thank you for your reply; I'm sorry I'm late in following up; I was busy recovering work and trying to patch code together from old backups.
I did contact the Support Team and am waiting for their reply.
Thanks again, 
-Mike

**Reply by queenofcorgis** [reply]
Thanks for reaching out to them @TrivediEnterprisesInc - hopefully you can get this squared away!

**Reply by windastra** [reply]
@TrivediEnterprisesInc Is there any updates for your account? sadly my account was compromised too, I have contacted Github team 8days ago, but still no answer though

**Comment by TrivediEnterprisesInc** [comment]
Unfortunately, no.  All three of my emails were hacked and so were my github repos (all pwds changed).
I lost nine months of work.
On the upside, though -- I gave myself 3 months to get back to where I was (since I coded it before I figured it would be easier this time).  And hey; I was back (around 90%) in ONE MONTH.
Good luck to you, and make sure you backup often.  I did but stopped coz I had so many email accounts; BUT that's no longer safe enough; you need more security.  I backup my stuff in encrypted zipfiles on various storage sites now.
Sorry again to hear about your bad luck, that's a real shame...  Good luck!
________________________________

**Comment by soujiokita** [comment]
So, basicaly i was Trojan/Generic.ASMalwS victims, all of my saved session user and password on Google Chrome were compromised to the Turkey Idiot actors, I have successfully retake all of my 21 account, but only missing one, my Github.

Hey @queenofcorgis can you tell the Github staff who handled my case,
I have same experience, but the bad actors changed my primary email, so I can't do nothing, I have contacted GitHub the case was confirmed as compromised so they suspend my hacked account, but again I have been waiting over a month still no further progress.

![Screenshot_49](https://user-images.githubusercontent.com/47995444/202924874-10c627db-3773-4a20-a0f2-1f96d6f36e12.png)

Github Team said: **"We have not forgotten about you."**

**Reply by queenofcorgis** [reply]
Hi there - do you have a ticket number and can you confirm the username of your taken account please - I can see what I can do!

**Comment by TrivediEnterprisesInc** [comment]
Hi, Lili!
My ticket # is  Ticket 1660456<https://support.github.com/ticket/1660456>.
Thank you,
-Mike
________________________________

**Reply by queenofcorgis** [reply]
Hi there - it looks like this ticket was closed about five months ago - was this not resolved?

**Comment by TrivediEnterprisesInc** [comment]
No, to the best of my knowledge it was not.  What I ended up doing was setting up a whole new slew of
email accounts and (I did have a backup but 9 months old!) sloggin to rebuild all missing code.

I am ALSO concerned about one further thing: this is not the first time this has happened to me (first on GitHub, though) and I feel that by locking me out of the TRACKING (i.e., commits) they might claim ownership of the code (they ARE thieves) and claim damages for ME stealing their code.
Does GitHub have a procedure for maintaining that kind of legal audit/document backtracking assurance?  Many firms I have worked at in the past did.
Thank you for your attention to this matter,
-Mike
________________________________

**Reply by queenofcorgis** [reply]
Hi Mike - I asked about your question regarding legal audit/document backtracking assurance, once I get an answer I'll update you. As for your ticket, I'd reference the last message sent from GitHub about the state of it.

**Reply by queenofcorgis** [reply]
@TrivediEnterprisesInc check out these links to answer your question: 
* [Reviewing Your Security Log](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/reviewing-your-security-log)
* [Reviewing the Audit Log for Your Organization](https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization)

**Comment by TrivediEnterprisesInc** [comment]
Hi, Lili!
Sorry for not responding sooner (I'm keeping logins @ a minimum in case I get hacked again!)
Thank you for the links you provided (Audit Log docs etc.) BUT since I no longer have access to my previous repos I can't see the audit logs either, you see!
Thanks again,
-Mike
________________________________

**Comment by PhongPhan1987** [comment]
***@***.***

Vào 22:22, Th 6, 9 thg 12, 2022 Mike Trivedi ***@***.***> đã
viết:
